rules/windows/execution_downloaded_shortcut_files.toml

```diff
@@ -36,15 +36,20 @@ sequence by process.entity_id with maxspan=2s
 
 [[rule.threat]]
 framework = "MITRE ATT&CK"
+
 [[rule.threat.technique]]
-id = "T1193"
+id = "T1566"
+name = "Phishing"
+reference = "https://attack.mitre.org/techniques/T1566/"
+[[rule.threat.technique.subtechnique]]
+id = "T1566.001"
 name = "Spearphishing Attachment"
-reference = "https://attack.mitre.org/techniques/T1193/"
+reference = "https://attack.mitre.org/techniques/T1566/001/"
 
-[[rule.threat.technique]]
-id = "T1192"
+[[rule.threat.technique.subtechnique]]
```

Comment:
can we have multiple sub techniques under one technique?

Reply:
yes